cli/jobs/pipelines-with-components/basics/4b_datastore_datapath_uri/pipeline.yml

```diff
@@ -8,7 +8,7 @@ inputs:
 outputs:
   pipeline_sample_output_data:
     # format for data store uri: azureml://datastores/<datastore_name>/paths/<path>/<on>/<datastore>
-    path: azureml://datastores/workspaceblobstore/paths/azureml  
+    path: azureml://datastores/workspaceblobstore/paths/azureml/set_path_on_datastore
```

Comment:
may use 4b_datastore_datapath_uri as the path here?

Reply:
updated